certified-connectors/AzureDigitalTwins/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1002 @@
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "result": {
+                  "type": "string",
+                  "description": "Results From the Twin.",
+                  "title": "Result"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get Twin by Id",
+        "description": "Pass in the twin id to retrieve its details.",
+        "operationId": "GetTwinById",
+        "parameters": [
+          {
+            "name": "twinid",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "api-version",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "default": "2020-10-31",
+            "x-ms-visibility": "internal"
+          }
+        ]
+      },
+      "delete": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "Delete Twin",
+        "description": "Deletes the Twin.",
+        "operationId": "DeleteTwin",
+        "parameters": [
+          {
+            "name": "twinid",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "api-version",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "default": "2020-10-31",
+            "x-ms-visibility": "internal"
+          }
+        ]
+      },
+      "put": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "Create new Twins",
+        "operationId": "AddTwin",
+        "parameters": [
+          {
+            "name": "twinid",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "api-version",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "default": "2020-10-31",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "Body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "value": {
+                  "type": "string",
+                  "description": "Value for the Twin.",
+                  "title": "Value"
+                }
+              }
+            }
+          }
+        ],
+        "description": "Creates a new Twin based on the message body."
+      },
+      "patch": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
```

Comment:
@sankumrav same here. Please look through all responses and define them according to the links in above comment.